runatlantis.io/docs/server-configuration.md

```diff
@@ -1070,6 +1080,16 @@ This is useful when you have many projects and want to keep the pull request cle
 
   The Redis Database to use when using a Locking DB type of `redis`. Defaults to `0`.
 
+### `--redis-deployment`
```

Comment:
Having the user have to specify this here and expect it to match const strings in redis package is a bit brittle.

Maybe we omit this entirely, and assume if the user specifies `--redis-cluster-addresses` they expect a cluster, otherwise they don't?

Otherwise we should validate that the input is one of these two choices, probably best to have an enum type in redis package